soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -514,10 +542,18 @@ void CheckTrackerTransition(uint32_t sceneNum) {
 }
 
 void CheckTrackerFrame() {
-    if (IS_RANDO) {
-        hideShopRightChecks = CVarGetInteger("gCheckTrackerOptionHideRightShopChecks", 1);
-        alwaysShowGS = CVarGetInteger("gCheckTrackerOptionAlwaysShowGSLocs", 0);
-    }
+    /*if (IS_RANDO) {
+        bool hideShopRightChecks_ = CVarGetInteger("gCheckTrackerOptionHideRightShopChecks", 1);
+        if (hideShopRightChecks != hideShopRightChecks_) {
+            hideShopRightChecks = hideShopRightChecks_;
+            RecalculateAreaTotals();
+        }
+        bool alwaysShowGS_ = CVarGetInteger("gCheckTrackerOptionAlwaysShowGSLocs", 0);
+        if (alwaysShowGS != alwaysShowGS_) {
+            alwaysShowGS = alwaysShowGS_;
+            RecalculateAreaTotals();
+        }
+    }*/
```

Comment:
now that this is working the other way can this part be removed instead of just commented out?

Reply:
Done.